src/components/Header/header.scss

```diff
@@ -0,0 +1,109 @@
+@import "/src/styles/styles.scss";
+.header {
+  &-container {
```

Comment:
why do you need your own container? 
we said earlier that we would use our general class grid-container for it)